scripts/core/services/pageTitle.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * PageTitle service
+ *
+ * PageTitle service is used to save and update the title of the page.
+ * The title will be 'Superdesk {- Url} {-Desk/Workspace}
+ * Url can be: Dashboard, Monitoring, Spike, Highlights, Search, Settings
+ * If Url is search then there won't be desk
+ *
+ */
+export default angular.module('superdesk.services.pageTitle', [])
+    .service('pageTitle', function() {
+        this.title = 'Superdesk';
+        this.url = '';
+        this.workspace = '';
+
+        this.setPageUrl = function(url) {
+            this.url = url;
+            setPageTitle(this.url, this.workspace);
+        };
+
+        this.setPageWorkspace = function(activeWorkspace) {
+            this.workspace = activeWorkspace;
+            setPageTitle(this.url, this.workspace);
+        };
+
+        function setPageTitle(url, activeWorkspace) {
+            document.title = 'Superdesk' + (url?' - ' + url:'') + (activeWorkspace?' - ' + activeWorkspace:'');
```

Comment:
I'm just wondering, would it make more sense for the title to be of the form: `Superdesk - <Workspace> - <Title>`? I'm fine with both, whatever makes more sense.